web/portal/brand/src/entities/Company/ForeignKeyGetter.tsx

```diff
@@ -62,6 +64,16 @@ export const foreignKeyGetter: ForeignKeyGetterType = async (
     });
   }
 
+  const isResidential = match.pathname.includes(Residential.localPath);
+  if (isResidential) {
+    promises[promises.length] = ResidentialFeatureSelectOptions({
+      callback: (options) => {
+        response.featureIds = options;
+      },
+      cancelToken,
+    });
+  }
```

Comment:
skip featureIds and add else condition (You're duplicating the request here and you probably have a race condition)